bridge/client/app/_components/ktb-project-settings-git/ktb-project-settings-git.component.ts

```diff
@@ -17,7 +17,11 @@ export class KtbProjectSettingsGitComponent implements OnInit {
   @Input()
   set gitData(gitData: GitData) {
     this.gitUrlControl.setValue(gitData.remoteURI || '');
+    this.gitUrlControl.markAsUntouched();
+    this.gitUrlControl.markAsPristine();
     this.gitUserControl.setValue(gitData.gitUser || '');
+    this.gitUserControl.markAsUntouched();
+    this.gitUrlControl.markAsPristine();
```

Comment:
Wrong control. Change it to `gitUserControl`.

Reply:
Yes :D